src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -1210,8 +1211,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Public Shared Function ClassifyTupleConversion(source As BoundTupleLiteral, destination As TypeSymbol, binder As Binder, <[In], Out> ByRef useSiteDiagnostics As HashSet(Of DiagnosticInfo)) As ConversionKind
+            If source.Type = destination Then
```

Comment:
entered a bug:   [https://github.com/dotnet/roslyn/issues/13373](https://github.com/dotnet/roslyn/issues/13373)


---
In reply to: [76156696](https://github.com/dotnet/roslyn/pull/13335#discussion_r76156696) [](ancestors = 76156696)